islands/HelloBar.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 the Deno authors. All rights reserved. MIT license.
+
+/** @jsx h */
+/** @jsxFrag Fragment */
+import { ComponentChildren, Fragment, h } from "preact";
+import { useEffect, useState } from "preact/hooks";
+import { tw } from "@twind";
+import { Cross } from "@/components/Icons.tsx";
+
+export default function HelloBar(props: {
+  to: string;
+  children: ComponentChildren;
+}) {
+  const [helloBar, setHelloBar] = useState(false);
+
+  useEffect(() => {
+    setHelloBar(!localStorage.getItem("helloBar"));
+  }, []);
+
+  return (
+    <div>
+      {helloBar
+        ? (
+          <div
+            class={tw`text-center bg-black text-white p-1 flex items-center justify-between flex-wrap`}
+          >
+            <div class={tw`flex-grow text-center`}>
+              <a
+                href={props.to}
+                target="_blank"
+                class={tw`inline-block p-1 hover:text-underline`}
+              >
+                {props.children}
+              </a>
+            </div>
+            <div
```

Comment:
make this a button and use the `IS_BROWSER` constant to disable it if it is false

Reply:
Why is the X button a `div` in other places? Also, is that `IS_BROWSER` required? It is an island, not a component.

Reply:
> Why is the X button a `div` in other places?

can you point to some examples?



> Also, is that `IS_BROWSER` required? It is an island, not a component.

because it is an island it should be used. The button doesnt do anything if its the SSR'd render, so we want to disable it until we get the client render

Reply:
https://github.com/denoland/dotland/blob/2544660f3a121a7686cb534b3a6dbd9767af557f/islands/GlobalSearch.tsx#L225

Reply:
huh, ig that should rather be a button